addon/templates/components/flexberry-maplayers.hbs

```diff
@@ -44,6 +44,7 @@
           settings=layer.settings
           visibility=layer.visibility
           layers=layer.layers
+          layer=layer
```

Comment:
@SeregaMatin as we have full layer model here, should we convert settings, visibility and layers to computed properties?